web-local/src/lib/components/Editor.svelte

```diff
@@ -309,9 +309,9 @@
 
   function underlineSelection(selections: any) {
```

Comment:
Better to type to get a sense of what `selections` is and understand the underlining code at a glance.

Reply:
Attempted to add the `SelectionRange` type and uncovered more issues. I've added a FIXME for now. This is not ideal but we'll need to go a bit further up the chain to resolve the type issues here.